### ISSUE TITLE
BUG TORNADOS : La Cumbre del Vórtice.

### ISSUE THREAD

**Issue by Alfonn** [RoDNet/Bugtracker-Infinium-4.0.6a#948]
Hice esta mazmorra ya 2 veces y nunca pudimos matar al último boss...
Simplemente porque cuando pasas del 1er boss cuando te vuelven a matar ya no puedes pasar de la 1er a la 2nda plataforma mediante 1 tornado que debería permanecer ahí..

![asd](https://f.cloud.github.com/assets/3854953/320283/be293dee-992d-11e2-951b-10c412d0798f.jpg)

![asd2](https://f.cloud.github.com/assets/3854953/320285/e6bbf972-992d-11e2-9acc-50f5bb83130d.jpg)

No se si me expliqué bien, pero las imágenes creo que lo dejan claro..

**Comment by crowel** [comment]
A nosotros nos mato el 2º boss el dragon y pasamos bien de nuevo, hace cosa de 15 días.

**Comment by Alfonn** [comment]
Sí, con el 2ndo boss sí, pero en el tercero ya los tornados desaparecen.

**Comment by Yavoy** [comment]
La primera vez que yo fui, se wipeo en el ultimo boss y al liberar, el ultimo tornado para llevarte a la ultima plataforma no estaba.. asi que no se como estara.

Hablo yo de hace unos 2 meses.
